examples/incremental_learning/helmet-detection-modelbox/incremental-learning-helmet-detection-modelbox.Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM modelbox/modelbox-develop-tensorflow_2.6.0-cuda_11.2-ubuntu-x86_64:latest
```

Comment:
1. What is the difference between this **_Dockerfile_** and another **_Dockerfile_** in examples directory? Are they duplicated?
2. Another question not about this Dockerfile, is **_incremental_learning-modelbox_** directory and files under it are duplicated compared to files in **_incremental_learning/helmet-detection-modelbox_** directory?